core/string/new_spec.rb

```diff
@@ -15,6 +15,12 @@
     end
   end
 
+  ruby_version_is "2.4" do
+    it "accepts a capacity argument" do
+      lambda { String.new("", capacity: 100_000) }.should_not raise_error
```

Comment:
Indeed, it's hard to know if this did anything.
`should_not raise_error` is the least useful matcher, could you replace it with `.should == ""` and
add an example where the first String argument is not empty?

Reply:
@eregon OK, done